### ISSUE TITLE
Flash fail using Bluejay on a working Blheli32 4 in 1 esc

### ISSUE THREAD

**Issue by warrenhdavis** [stylesuxx/esc-configurator#477]
I tried to flash a SKYSTARS H743 DUAL BMI270 GYRO FLIGHT CONTROLLER AND KM60 60A BLHELI-32 ESC STACK. It failed in Bluejay and the ESC’s say they are corrupted in Blheli32 now and of course I cannot reflash them now on either Bluejay or Blheli.

**Comment by fberggoetz** [comment]
Blujay is not for 32 bit esc's. And afaik the km60 is an am32 ESC anyways. Blujay is only for blheli_s  esc's.
Try https://am32.ca/ for the km60. That's what I use on mine and what it came with.

**Comment by stylesuxx** [comment]
What @fberggoetz said. How were you even able to flash something on that ESC with esc-configurator? Please post the log.

**Comment by ahmad-fayyaz** [comment]
In the same boat unfortunately, accidentally tried flashing Bluejay on Redfox A3 AIO on a 3 inch. Forgot it had Blheli_32. Tried to reflash using the BLHELISUITE but getting issues. Couldn't get log from esc_confgurator but it seemed to have corrupted the ESCs. Surprisingly ESCs are functional, however, you cannot change any settings so its basically locked. The startup tune disappeared and cannot write melody editor changes. I've attached images from BLHELISUITE which might help. 

<img width="914" height="732" alt="Image" src="https://github.com/user-attachments/assets/f81fa3af-be1b-48eb-87f8-e16aa90d3d48" />
<img width="914" height="732" alt="Image" src="https://github.com/user-attachments/assets/0951e24c-4dbe-44a2-8cf1-cc1b20e7d8a6" />
